cmd/podman/common.go

```diff
@@ -524,7 +524,7 @@ var createFlags = []cli.Flag{
 	},
 	cli.StringFlag{
 		Name:  "workdir, w",
-		Usage: "Working `directory inside the container",
```

Comment:
this should be amended to the commit that introduced the change

Reply:
Umm... do you want me to squash all the patches into one?

Reply:
Yes

Reply:
Done.